src/configs/Config.ts

```diff
@@ -76,5 +76,11 @@ export const config = {
       "primary_key": "id",
       "references": []
     }
-  }
-}
\ No newline at end of file
+  },
+  "exhaust_provider": process.env.exhaust_provider || "aws", // Supported providers - AWS, GCP, Azure
+  "exhaust_region": process.env.exhaust_region || "", // Region for the cloud provider storage
+  "exhaust_config": process.env.exhaust_config ? JSON.parse(process.env.exhaust_config) : {}, // Respective credentials object for cloud provider
+  "label_container": process.env.label_container || "", // Storage container/bucket name
+  "label_container_prefix": process.env.label_container_prefix || "", // Path to the folder inside container/bucket. Empty if data at root level
+  "storage_url_expiry": process.env.storage_url_expiry ? parseInt(process.env.storage_url_expiry) : 3600, // in seconds, Default 1hr of expiry for Signed URLs.
```

Comment:
I would suggest you to define one exhaust object and list all the properties which are required for the exhaust API.